Vod Review/ApexVod_UI.py

```
@@ -0,0 +1,254 @@
+        logging.getLogger().addHandler(logTextBox)
+        logging.getLogger().setLevel(logging.DEBUG)
+
+        self.grid.addWidget(logTextBox.widget, 8, 0, 1, 3)
+
+        # button box
+        self.buttonBox = QtGui.QDialogButtonBox(QtCore.Qt.Horizontal, self)
+        self.buttonBox.setStandardButtons(QtGui.QDialogButtonBox.Cancel | QtGui.QDialogButtonBox.Ok)
+        self.buttonBox.button(QtGui.QDialogButtonBox.Cancel).setEnabled(False)
+        self.grid.addWidget(self.buttonBox, 9, 1)
+        
+        self.buttonBox.button(QtGui.QDialogButtonBox.Ok).clicked.connect(self.run_code)
+
+        self.setLayout(self.grid)
+
+        # self.statusBar()
+        self.setWindowTitle('Apex Vod Review')
+
+        # QtGui.QApplication.setStyle(QtGui.QStyleFactory.create('Cleanlooks'))
+
+        self.show()
+
+    def center(self):
+        """ centers the window to the center of the screen when opened """
+        frameGm = self.frameGeometry()
+        screen = QtGui.QApplication.desktop().screenNumber(QtGui.QApplication.desktop().cursor().pos())
+        centerPoint = QtGui.QApplication.desktop().screenGeometry(screen).center()
+        centerPoint.setX(centerPoint.x() - frameGm.width() / 2)
+        centerPoint.setY(centerPoint.y() - frameGm.height() / 2)
+        frameGm.moveTo(centerPoint)
+        self.move(frameGm.topLeft())
+
+    def closeEvent(self, event):
+        """ spawns a pop up to check if you intended to exit """
+
+        reply = QtGui.QMessageBox.question(self, 'Message',
+                                           "Are you sure you want to quit?", QtGui.QMessageBox.Yes |
+                                           QtGui.QMessageBox.No, QtGui.QMessageBox.No)
+
+        if reply == QtGui.QMessageBox.Yes:
+            event.accept()
+        else:
+            event.ignore()
+
+    def selectFileInput(self):
+        """ File selection pop up for input """
+        fname = QtGui.QFileDialog.getOpenFileName(self, 'Select file', 'C:\\')

```

From memory you can set some sort of default filter so it will automatically hide files that aren't within a list of file extensions. So you can set it to auto show mp4, m4v, mkv, etc. and hide shit like docx, mp3, txt, etc.